nmrfx-core/src/main/java/org/nmrfx/chemistry/relax/OrderPar.java

```diff
@@ -287,11 +286,11 @@ public Double getValue() {
     public Double getError() {
         return error;
     }
-//    static final String[] PAR_NAMES = {"Order_param", "Tau_e", "Tau_f", "Tau_s", "Rex", "Sf2", "Ss2"};
 
     @Override
     public Double getValue(String name) {
         switch (name) {
+            case "Order_param":
             case "S2":
                 return value;
```

Comment:
If "Order_param" and "S2" are both returning the same value I think you can write it like this
```suggestion
            case "Order_param", "S2":
                return value;
```

Reply:
Oh actually this comma separated format was only available from java 14 on, so I don't know if we can make this change yet?

Reply:
Yes, I haven't started moving to post 11 features